src/Bundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -135,4 +136,68 @@ private function loadResources(array $loadedResources, ContainerBuilder $contain
             }
         }
     }
+
+    private function createTranslationParameters(array $config, ContainerBuilder $container): void
+    {
+        $this->createEnabledLocalesParameter($config, $container);
+        $this->createDefaultLocaleParameter($config, $container);
+    }
+
+    private function createEnabledLocalesParameter(array $config, ContainerBuilder $container): void
+    {
+        $enabledLocales = $config['translation']['enabled_locales'];
+
+        if (count($enabledLocales) > 0) {
+            $container->setParameter('sylius.resource.translation.enabled_locales', $enabledLocales);
+
+            return;
+        }
+
+        if ($container->hasParameter('locale')) {
+            $container->setParameter('sylius.resource.translation.enabled_locales', [$container->getParameter('locale')]);
+
```

Comment:
Here's where a deprecation notice should be triggered warning users that this parameter will no longer used in 2.0 and they should set the `enabled_locales` option though configuration.

Reply:
Done